code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -79,17 +79,17 @@
 	max_shells = 6
 
 // bwoincognito:Tasald Corlethian
-/obj/item/weapon/gun/projectile/revolver/detective/fluff/tasald_corlethian
+/obj/item/weapon/gun/projectile/revolver/mateba/fluff/tasald_corlethian //Now that it is actually Single-Action and not hacky broken SA, I see no reason to nerf this down to .38. --Joan Risu
 	name = "Big Iron revolver"
-	desc = "A .38 revolver for veteran rangers on the planet Orta. The right side of the handle has a logo for Quarion industries, and the left is the Rangers. The primary ammo for this gun is .38 rubber. According to the CentCom Chief of Security, this revolver was more controversial than it needed to be."
+	desc = "A .357 revolver for veteran rangers on the planet Orta. The right side of the handle has a logo for Quarion industries, and the left is the Rangers. The primary ammo for this gun is .38 rubber. According to the CentCom Chief of Security, this revolver was more controversial than it needed to be."
 
 	icon = 'icons/vore/custom_guns_vr.dmi'
 	icon_state = "tasaldrevolver"
 
 	item_state = "revolver"
 
 	fire_sound = 'sound/weapons/pistol.ogg'
-	ammo_type = /obj/item/ammo_casing/c38r
+	ammo_type = /obj/item/ammo_casing/a357/rubber //Like I said, no reason to nerf. --Joan Risu
```

Comment:
Aro rewrote the fluff .38 to be single action with an additinal fire delay. 

I approved the original app but while it was pending we couldn't get the original code to work without being hacky which broke things hence the .38 nerf. SA code works now so no need for nerf.